portals/publisher/src/main/webapp/source/src/app/components/Apis/Details/Policies/PoliciesSection.tsx

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import {
+    Grid, makeStyles, Typography,
+} from '@material-ui/core';
+import React, { FC } from 'react';
+import Box from '@material-ui/core/Box';
+import CONSTS from 'AppData/Constants';
+import { isRestricted } from 'AppData/AuthManager';
+import OperationPolicy from './OperationPolicy';
+import OperationsGroup from './OperationsGroup';
+import type {Policy, PolicySpec } from './Types';
+import PoliciesExpansion from './PoliciesExpansion';
+import MuiAlert from 'AppComponents/Shared/MuiAlert';
+import { FormattedMessage } from 'react-intl';
+import Alert from '@material-ui/lab/Alert';
+
+
+
+
+const useStyles = makeStyles((theme:any) => ({
+    gridItem: {
+        display: 'flex',
+        width: '100%',
+    },
+    alert: {
+        backgroundColor: 'transparent'
+    }
+}));
+
+interface PolicySectionProps {
+    openAPISpec: any;
+    isChoreoConnectEnabled: boolean;
+    isAPILevelGranularitySelected: boolean;
+    allPolicies: PolicySpec[] | null;
+    policyList: Policy[];
+    api: any;
+    expandedResource: string | null;
+    setExpandedResource: React.Dispatch<React.SetStateAction<string | null>>;
+
+}
+
+/**
+ * Renders the policy management page.
+ * @returns {TSX} Policy management page to render.
+ */
+const PoliciesSection: FC<PolicySectionProps> = ({
+    openAPISpec,
+    isChoreoConnectEnabled,
+    isAPILevelGranularitySelected,
+    allPolicies,
+    policyList,
+    api,
+    expandedResource,
+    setExpandedResource,
+}) => {
+    const classes = useStyles();
+    let borderColor = "";
+
+    return (
+        <Box>
+            {isAPILevelGranularitySelected ? (
+                <Box m={1} p={0.1} mt={1.5}
+                    sx={{ boxShadow: 0.5, bgcolor: borderColor, borderRadius: 1 }}
+                >
+                    <Alert severity="info" className={classes.alert}>
+                        API level policies will execute before Operation Level Policies
+                    </Alert>
```

Comment:
IMO this alert takes up too much space in the UI. Given that the same alert is repeated under both tabs, moving it to a commonly visible UI block would be better. @chanaka3d any thoughts on this?

If there is no other place that we can move this to, at least let's shrink the spacing available at the top and bottom of it.